src/mappings/entities/mapClaim.ts

```diff
@@ -25,6 +22,11 @@ type ClaimParams = {
   jurisdiction: string;
 };
 
+type Scope = {
```

Comment:
(nit) I think interface is preferred, although it does almost the same thing as a type.